my-workshop/public/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="icon" type="image/x-icon" href="favicons/favicon.ico">
+    <meta charset="UTF-8" />
+    <title>Round Rock Plumber</title>
+    <link rel="stylesheet" href="styles/styles.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <header>
+      <div class="img-container">
+        <img class="header-photo" src="img/262012-P4LQCB-641 1.png" alt="Error" />
+        <img class="img-plumbers" src="img/Plumbers.png" alt="Error" />
+        <div class="p-container">
+          <p class="round-rock">ROUND ROCK</p>
+          <p class="plumbing">PLUMBING</p>
+        </div>
+        <div class="lorem-container">
+          <h1 class="h1-lorem-ipsum-header">What is Lorem Ipsum</h1>
+          <p class="p-lorem-ipsum-header">
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled
+          </p>
+        
+          </div>
+      <div class="wrench-rectangle-container">
+        <img class="wrench-1" src="img/wrench 1 (1).png" alt="Error" />
+        <img class="rectangle-1" src="img/Rectangle 1 (1).png" alt="Error" />
+
+        <img class="wrench-2" src="img/wrench 1 (1).png" alt="Error" />
+        <img class="rectangle-2" src="img/Rectangle 1 (1).png" alt="Error" />
+
+        <img class="wrench-3" src="img/wrench 1 (1).png" alt="Error" />
+        <img class="rectangle-3" src="img/Rectangle 1 (1).png" alt="Error" />
+        <p class="p-services-1">Services one</p>
```

Comment:
it's a pity that you didn't use Bootstrap
it's very powerful tool so I recommend you to study and use it